c2c/code-to-cloud-spec.md

```diff
@@ -281,27 +293,29 @@ Contains the secrets required for the application.
 
 Contains external files required for the code.
 
-|Identifier |Description                                        |Example Value                      |
-|---	    |---	                                            |---	                              |
-|file    |Path of the external file            |"resource/file.txt"          |
-|mount_path |Path of the file in the container    |"/home/ballerina/foo/file.txt" |
+|Identifier |Description                                        |Example Value     |
+|---	    |---	                                            |---	           |
+|file    |Path of the external file            |"resource/file.txt"                |
+|mount_path |Path of the file in the container    |"/home/ballerina/foo/file.conf" |
 
 #### [[cloud.config.files]]
 
 Contains config toml files required for the code.
 
-|Identifier |Description                                        |Example Value                      |
-|---	    |---	                                            |---	                              |
-|file    |Path of the external file            |"conf/Config.toml"          |
+|Identifier |Description                                        |Example Value   |
+|---	    |---	                                            |---	         |
+|file    |Path of the external file            |"conf/Config.toml"               |
+|name    |Name field for the configmap            |"sample5-config-map"          |
 
 #### [cloud.deployment]
 
 Contains the properties related to the deployment.
 
-|Identifier            |Description                                             |Default Value    |
+|Identifier            |Description                                              |Default Value     |
 |---	                |---	                                                 |---	            |
-|internal_domain_name   |Name of the internal domain (k8s service name)         |${MODULE_NAME}-svc|
-|min_memory            |Minimum memory allocated to the container             |"100Mi"        |
+|internal_domain_name   |Name of the internal domain (k8s service name)          |${MODULE_NAME}-svc|
+|external_accessible   |External accessibility of the deployment                 |                  |
```

Comment:
we need to specify the default value, I think the value is false, please recheck with code